.eslintrc.json

```diff
@@ -0,0 +1,12 @@
+{
+    "env": {
+        "browser": true,
+        "es2020": true
+    },
+    "parserOptions": {
+        "ecmaVersion": 11,
+        "sourceType": "module"
+    },
+    "rules": {
+    }
+}
```

Comment:
Should we also add [eslint-config-prettier](https://github.com/prettier/eslint-config-prettier) and add it to the workflow?
We also need to add `extends`:
```
{
  "extends": [
    "some-other-config-you-use",
    "prettier"
  ]
}
```

Reply:
What does it do? Linting _and_ formatting?

Reply:
> 
> 
> What does it do? Linting _and_ formatting?

According to official documentation, it:
> Turns off all rules that are unnecessary or might conflict with Prettier.